VoiceAttackResponder/VoiceAttackPlugin.cs

```diff
@@ -675,9 +675,11 @@ public static void InvokeSay(ref dynamic vaProxy)
 
                 string voice = vaProxy.GetText("Voice");
 
+                int? volume = vaProxy.GetInt("Volume") ?? 100;
```

Comment:
If `transmitVolume` wasn't dependent on `Configuration.Volume`, we could simply allow this too be null when it wasn't set by the player.